### ISSUE TITLE
new feature for quiz app

### ISSUE THREAD

**Issue by spirogg** [PHPfox-Official/phpfox-v4-feature-requests#864]
when creating a quiz, I would like to have an area to add a Description of the answer - additional information 
- So when someone submits their answer there will be a description of the answer also

basically if I ask a question for the quiz I can add my answer then I can add a description of the answer to an area in the quiz app

see photos below for my Idea I saw elsewhere

would be cool Idea if you can make quiz like this

1) 
- 
![Capture](https://user-images.githubusercontent.com/17736688/63254689-c83ee400-c239-11e9-9a80-5300ea3cb6ca.PNG)

2) 
- 
![Capture1](https://user-images.githubusercontent.com/17736688/63254697-ce34c500-c239-11e9-8b0b-6eaba6aa3a43.PNG)

3) 

- 
![Capture2](https://user-images.githubusercontent.com/17736688/63254709-d4c33c80-c239-11e9-9062-b09595aac320.PNG)

this would be awesome for quiz app

thanks
Spiro

**Comment by harrison05** [comment]
Yes, I think it's awesome, but I have to check with my team. I will inform you if have any update.

Regards.
